gradle.properties

```diff
@@ -1,5 +1,5 @@
 # Here is where we manage the version
-releaseVersion=0.3.1-SNAPSHOT
+releaseVersion=0.3.1
```

Comment:
this scares me because we haven't actually tested that this reports correctly

Reply:
This is only intended to fix up the manifest, shouldn't have any functional changes (except it might also fix #65 )